caiman/gui/gui_pyqtgraph_layout.py

```diff
@@ -0,0 +1,323 @@
+import pyqtgraph as pg
+from pyqtgraph import FileDialog
+from pyqtgraph.Qt import QtCore, QtGui
+from pyqtgraph.parametertree import Parameter, ParameterTree, ParameterItem, registerParameterType
+import numpy as np
+import caiman as cm
+from caiman.source_extraction.cnmf.cnmf import load_CNMF
+from caiman.source_extraction.cnmf.online_cnmf import load_OnlineCNMF
+
+import cv2
+import scipy
+import os
+# Always start by initializing Qt (only once per application)
+app = QtGui.QApplication([])
```

Comment:
This file has a lot of code scattered about not living in functions that will run on import. Might be better to move all that to an initialisation function that people need to call.